hu/admin/nests.php

```diff
@@ -25,7 +25,7 @@
     ],
     'variables' => [
         'notices' => [
-            'variable_deleted' => 'A változó ":variable" törölve lett, és az újjáépítés után már nem lesz elérhető a szerverek számára.',
+            'variable_deleted' => 'A változó ":variable" törölve lett, és az újjáépítés után már nem lesz elérhető a többi szerver számára.',
```

Comment:
Inkább ezt a szókapcsolatot használnám az ehhez hasonló mondatoknál:
```suggestion
            'variable_deleted' => 'A(z) ":variable" változó törölve lett, és az újjáépítés után már nem lesz elérhető a többi szerver számára.',
```